department-of-veterans-affairs/caseflow
# GitHub Action: Push `make docs` files to wiki

### Description
Add a GitHub Action that runs `make docs` and puts the artifact into the wiki whenever a PR is merged. Wiki pages can then link to those files. This will allow us to (1) remove the `make docs` instruction in the PR template and (2) remove all the files in docs/schema (including those large svg files) from our main repo's codebase.

From a wiki page, the files can be referenced like:
* https://raw.githubusercontent.com/wiki/department-of-veterans-affairs/caseflow/make_docs/schema/caseflow-subclasses.svg
* https://raw.githubusercontent.com/wiki/department-of-veterans-affairs/caseflow/make_docs/schema/caseflow.csv

For example, https://github.com/department-of-veterans-affairs/caseflow/wiki/Caseflow-Database-Schema-Documentation has been updated to use these files.

### Acceptance Criteria

### Testing Plan
- Go to "Actions" tab at the top of this page (https://github.com/department-of-veterans-affairs/caseflow/actions)
- Check that the last 'Make-docs-to-wiki' action ran successfully (Note: it only takes 2 minutes to run!)
- Check for the files in the wiki github repo by cloning `https://github.com/department-of-veterans-affairs/caseflow.wiki.git`
